pupil-spa/src/assets/govuk-elements-sass/elements/_forms.scss

```diff
@@ -0,0 +1,239 @@
+
```

Comment:
@GeorgeChatzigiannis compilation fails with inline comments so the migration scripts (that need more work) are removing them resulting in these extra lines. I'll fix this as part of the pending work.